addons/website/static/src/interactions/popup/no_backdrop_popup.js

```diff
@@ -0,0 +1,79 @@
+import { registry } from "@web/core/registry";
+import { isScrollableY } from "@web/core/utils/scrolling";
+import { Interaction } from "@website/core/interaction";
+
+export class NoBackdropPopup extends Interaction {
+    static selector = ".s_popup_no_backdrop";
+    // TODO: handle edit mode
+    // disabledInEditableMode = false;
+    dynamicContent = {
+        "_root": {
+            "t-on-shown.bs.modal": this.onModalNoBackdropShown,
+            "t-on-hide.bs.modal": this.removeModalNoBackdropEvents,
+        }
+    };
+
+    setup() {
+        this.throttledUpdateScrollbar = this.throttledForAnimation(this.updateScrollbar);
+        // TODO: handle edit mode
+        // if (this.editableMode && this.el.classList.contains("show")) {
+        //     // Use case: When the "Backdrop" option is disabled in edit mode.
+        //     // The page scrollbar must be adjusted and events must be added.
+        //     this.updateScrollbar();
+        //     this.addModalNoBackdropEvents();
+        // }
+    }
+
+    destroy() {
+        this.removeModalNoBackdropEvents();
+        // After destroying the interaction, we need to trigger a resize event
+        // so that the scrollbar can adjust to its default behavior.
+        window.dispatchEvent(new Event("resize"));
+    }
+
+    updateScrollbar() {
+        // When there is no backdrop the element with the scrollbar is
+        // '.modal-content' (see comments in CSS).
+        const modalContentEl = this.el.querySelector(".modal-content");
+        const isOverflowing = isScrollableY(modalContentEl);
+        const bsModal = window.Modal.getInstance(this.el);
+        if (isOverflowing) {
+            // If the "no-backdrop" modal has a scrollbar, the page's scrollbar
+            // must be hidden. This is because if the two scrollbars overlap, it
+            // is no longer possible to scroll using the modal's scrollbar.
+            bsModal._adjustDialog();
+        } else {
+            // If the "no-backdrop" modal does not have a scrollbar, the page
+            // scrollbar must be displayed because we must be able to scroll the
+            // page (e.g. a "cookies bar" popup at the bottom of the page must
+            // not prevent scrolling the page).
+            bsModal._resetAdjustments();
+        }
+    }
+
+    addModalNoBackdropEvents() {
```

Comment:
maybe add the updateScrollbar call here, and then you can remove the onModalNoBackdropShown method, i think